proto/backend.proto

```diff
@@ -306,3 +306,160 @@ message StreamPacket {
   // JSON-encoded data to publish into a channel.
   bytes data = 1;
 }
+
+//-----------------------------------------------------------------
+// Storage -- EXPERIMENTAL and is subject to change until 12.??
+//-----------------------------------------------------------------
+
+service Storage {
```

Comment:
This is a service that a plugin would implement, right?

Would AdmissionControl be a better service name? `Storage` makes me think it should have read and write operations.

Reply:
yes -- i'll update